00-introduction.md

```diff
@@ -21,6 +21,7 @@ This is version 0.
 7. [BOLT #7](07-routing-gossip.md): P2P Node and Channel Discovery
 8. [BOLT #8](08-transport.md): Encrypted and Authenticated Transport
 9. [BOLT #9](09-features.md): Assigned Feature Flags
+11. [BOLT #11](11-payment-encoding.md): Invoice Protocol for Lightning Payments
```

Comment:
@JaredR26 : Please do not spread misinformation. BU is not a system developed on top of Bitcoin, it is a Bitcoin client, as is Classic. Just because these clients propose alterations to the consensus rules for blocks > 1MB does not mean they are not part of Bitcoin.